react/src/quiz/quiz-components.tsx

```diff
@@ -71,11 +71,16 @@ export function DownloadUpload(): ReactNode {
     return (
         <div style={{ marginTop: '5px' }}>
             <button onClick={() => { exportQuizPersona() }}>
-                Download Quiz History
+                Export Quiz History
             </button>
             {' '}
             <button onClick={() => { void importQuizPersona() }}>
-                Upload Quiz History
+                Restore Quiz History
```

Comment:
@kavigupta How do we feel about terminology here?

Reply:
I agree with @ealgase  that if we're merging Upload isn't great, but if it's Export it should be Import

Reply:
Switched it to Import.